packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -98,6 +99,12 @@ export class StripeWebhookHandler extends StripeHandler {
         case 'invoice.payment_failed':
           await this.handleInvoicePaymentFailedEvent(request, event);
           break;
+        case 'product.updated':
+          await this.handleProductUpdate(request, event);
+          break;
+        case 'plan.updated':
+          await this.handlePlanUpdate(request, event);
```

Comment:
nit: the other event handling functions all have names in the format of handle*Event